semgrep/semgrep/semgrep_main.py

```diff
@@ -87,16 +77,17 @@ def validate_configs(
         rules = config.get(RULES_KEY)
         valid_rules = []
         invalid_rules = []
-        for rule in rules:  # type: ignore
-            if validate_single_rule(config_id, rule):
+        for rule_dict in rules:  # type: ignore
```

Comment:
do we still have to ignore type here?

Reply:
I think so because an unused ignore is an error